### DISCUSSION TITLE
Label Description for Experiment Group

### DISCUSSION THREAD

**Discussion by nirajgiriXD** [privacysandbox/privacy-sandbox-dev-support#268] [General] [answered]
Chrome is set to introduce a new header `Sec-Cookie-Deprecation` containing `label` as the value which will allow websites to identify the specific `experiment group` a browser belongs to. I've been searching for information on these labels, resembling something like the table provided, but couldn't find anything. Is there a recommended site or resource where one can gain insight into these labels?

| **Label** | **% of browser** | Description |PS R&M APIs | 3PC |
| ---- | ---- | ---- | ---- | ---- |
| `control_1.1` | 0.25 |  | Disabled | Enabled |
| `control_1.2` | 0.25 |  | Disabled | Enabled |
| `control_1.3` | 0.25 |  | Disabled | Enabled |
| `control_1.4` | 0.25 |  | Disabled | Enabled |
| `control_2` | 0.25 |  | Disabled | Disabled |
| `label_only_1` | 1.5 |  |  |  |
| `label_only_2` | 1.5 |  |  |  |
| `label_only_3` | 1.5 |  | Enabled | Disabled |
| `label_only_4` | 1.5 |  |  |  |
| `label_only_5` | 1.5 |  |  |  |
| `treatment_1.1` | 0.25 |  |  |  |
| `treatment_1.2` | 0.25 |  |  |  |
| `treatment_1.3` | 0.25 |  |  |  |

`Note`: I couldn't find any descriptions of these labels in the [Chrome-facilitated testing](https://developers.google.com/privacy-sandbox/setup/web/chrome-facilitated-testing).

**Comment by samdutton** [comment]
Hi @nirajgiriXD — the meaning of the labels is described in the Mode A and Mode B sections of the [Chrome-facilitated testing](https://developers.google.com/privacy-sandbox/setup/web/chrome-facilitated-testing) documentation. 

Your table looks like a useful way to present this information. We may well provide something like this!

I'll close #269, since it seems to be a duplicate of this one.

**Comment by nirajgiriXD** [comment]
@samdutton Thank you for your quick response. I have thoroughly reviewed the [Chrome-facilitated testing](https://developers.google.com/privacy-sandbox/setup/web/chrome-facilitated-testing) documentation and created the table based on the information I gathered from it. However, the documentation does not provide information about individual labels. For example, what does the `treatment_1.1` label mean?

> Your table looks like a useful way to present this information. We may well provide something like this!

Having the table with additional fields would be really helpful 🙂.

Thank you once again for your assistance. I look forward to any further updates.

**Comment by samdutton** [comment]
Hi @nirajgiriXD 

> the documentation does not provide information about individual labels. For example, what does the treatment_1.1 label mean?

These labelled experiment arms are designed to allow ad techs flexibility in combining labels to create their own experiments of varying sizes. 

The purpose of the labels is outlined in the [guidance on industry testing](https://www.gov.uk/cma-cases/investigation-into-googles-privacy-sandbox-browser-changes#industry-testing) from the UK [Competition and Markets Authority](https://www.gov.uk/cma-cases/investigation-into-googles-privacy-sandbox-browser-changes#industry-testing). For example, `Treatment` labels are intended for "impressions served without using data related to third-party cookies", the `Control 1` labels for "impressions served with data related to third-party cookies and removing data related to new Privacy Sandbox APIs".

**Comment by nirajgiriXD** [comment, marked as answer]
Hi @samdutton, I reviewed the contents of the following two documents:
- [Chrome-facilitated testing](https://developers.google.com/privacy-sandbox/setup/web/chrome-facilitated-testing)
- [Relevance APIs testing guidance](https://developers.google.com/privacy-sandbox/setup/web/relevance-testing)

Based on the information gathered, I compiled the table below. Could you confirm if the table data is accurate?

Label | Mode | PS R&M APIs | 3PC | Description
-- | -- | -- | -- | --
control_1.* | A | Disabled | Enabled | Status Quo: Representing normal Chrome traffic without using Topics API or Protected Audiences auctions.
label_only_* | A | Enabled | Disabled | Allows for coordinated experiments across different ad techs without third-party cookies, using PS R&M APIs.
treatment_1.* | B | Enabled | Disabled | Represents browsers with third-party cookies disabled but Privacy Sandbox APIs available.
control_2 | B | Disabled | Disabled | Used where both third-party cookies and the PS R&M APIs are disabled, serving as a control group.

**Comment by samdutton** [comment]
Thanks @nirajgiriXD — table looks accurate. Might be worth reiterating for `control_1.*` and `label_only_*` that assignment to the experiment arms that get these labels does not affect functionality.
